src/components/WebexAvatar/WebexAvatar.jsx

```diff
@@ -43,8 +35,15 @@ export default function WebexAvatar({
   style,
 }) {
   const {avatar, displayName, status} = usePerson(personID);
+  const [randomPlaceholder] = useState(
+    () => Math.floor(Math.random() * numberOfPlaceholders) + 1,
+  );
   const type = displayStatus ? status : null;
-  const cssClasses = webexComponentClasses('avatar', className);
+  const initials = displayName?.split(' ').map((name) => name.charAt(0)).slice(0, 2);
+  const placeholderClassName = `placeholder-${randomPlaceholder}`;
+  const cssClasses = webexComponentClasses('avatar', className, undefined, {
+    [placeholderClassName]: !avatar && displayName,
```

Comment:
i think you can always put this placeholder class

Reply:
This is to prevent the avatar to have a placeholder when we have an image because it looks like the avatar has an outline.
![image](https://user-images.githubusercontent.com/57891813/131352086-3f36fe4f-4746-460f-a882-79c6e7e401f5.png)